### PR TITLE
Provide additional capabilities around schema writing

### PR DESCRIPTION
**Pull request by josephschorr**

### CODE REVIEW

File: internal/services/shared/schema_test.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	"github.com/authzed/spicedb/pkg/datastore"
+
+	"github.com/authzed/spicedb/pkg/schemadsl/input"
+
+	"github.com/authzed/spicedb/pkg/schemadsl/compiler"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/authzed/spicedb/internal/datastore/memdb"
+	"github.com/authzed/spicedb/internal/testfixtures"
```
Comment (jzelinskie) [inline internal/services/shared/schema_test.go:16]: These imports are grouped wrong.

**Review by jzelinskie — CHANGES_REQUESTED** [review]
only a nitpick

File: internal/services/shared/schema_test.go (hunk 1 above)
Comment (josephschorr) [reply internal/services/shared/schema_test.go:16]: Damned it... VSCode strikes again

**Comment by josephschorr** [comment]
Updated

**Review by jzelinskie — APPROVED** [review]

**Pull request merged**